packages/opentelemetry-context-async-hooks/src/AsyncHooksContextManager.ts

```diff
@@ -103,7 +101,7 @@ export class AsyncHooksContextManager implements ContextManager {
 
   private _bindFunction<T extends Function>(target: T, context: Context): T {
     const manager = this;
-    const contextWrapper = function(this: {}, ...args: unknown[]) {
+    const contextWrapper = function (this: {}, ...args: unknown[]) {
```

Comment:
are we sure we want to have this extra space here ?,

Reply:
it's matter of being consistant. Sometime we do, sometime we don't. By default, gts require this rule, so I prefer to let it in order to be neutral. Finally, for what I see extra space seems to be more used overall.

Reply:
We have already enforced this previously to be without space, now this is going to look like a change not connected to replacing tslint with eslint. I think we should be consistent - doesn't matter if we use tslint eslint or something different, the rules should remain the same, only the mechanism which checks that should be replaced.

Reply:
Do you want to deactivate the rule ?

Reply:
notice that it's a gts breaking change

Reply:
hmm not sure if I understand correctly. We had rule previously which kept no space, isn't there a similar rule ?  And if you deactivate this rule you mentioned does it mean the both cases will be possible or ?

Reply:
I made the bad assumption that prettier was configured through eslint in order to have one configuration file but it isn't the case with gts and I need to have a prettier config file that inherits from gts plugin... So by default we have the default rules. I'm fixing this. (gts doesn't care if there is a space or not)

Reply:
gts add extra space too

Reply:
and it seems to be worst with the prettierrc.json file through the gts plugin
https://github.com/open-telemetry/opentelemetry-js/issues/1083